storage/stats.go

```diff
@@ -51,6 +51,13 @@ func newRangeStats(rangeID roachpb.RangeID, e engine.Engine) (*rangeStats, error
 	return rs, nil
 }
 
+// Replace *rs with the contents of *other.
+func (rs *rangeStats) Replace(other *rangeStats) {
+	rs.Lock()
+	defer rs.Unlock()
+	rs.MVCCStats = other.MVCCStats
```

Comment:
Yeah, I guess so. There's a theoretical deadlock risk if you lock both of them but we don't need to worry about it.